draft-ietf-httpbis-digest-headers.md

```diff
@@ -592,9 +593,15 @@ Digest: sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
 "hello"
 ~~~
 
-### Digest in both Request and Response. Returned value depends on representation metadata
+### Digest in both Request and Response. Field-values depends on representation metadata
+
+The request contains a `Digest` header
+calculated on the enclosed representation.
```

Comment:
Here I would highlight the `Accept-Encoding` header and then reword the next paragraph slightly e.g. 

"It also includes an `Accept-Encoding: br` header field that advertises the client supports brotli encoding"